### DISCUSSION TITLE
Pop-up/ad/redirect blocker in Web View

### DISCUSSION THREAD

**Discussion by Konkss** [aniyomiorg/aniyomi#882] [Ideas]
I'm someone that likes to check in on the comments sections of episodes, and many host sites are almost unusable in the web view.

Essentially, some/most sites have aggressive popup and full-page click ad redirects that take over the web view address, so I'm constantly mashing the back arrow in hopes I can get back to the series/episode page.

I'm hoping there could be a simple blocker that can be implemented into the web view that'll make it less chaotic. I'm not asking for a full fat ad-blocker. Banner ads aren't the issue. It's just the aggressive page re-directs that, imo, need to be dealt with.

anyone else encounter this when they're trying to peep some comments?
It's like I'm back in in the 2000's again.

Or if there was a way to have a little interface section to host an extensions Disqus comments section... but I feel like that's much harder to implement than a blocker.

**Comment by 002UwU-fan** [comment]
i just discovered that there's an ad blocker built-in

1-go to settings
2-advanced
3-dns over https
4-ad-guard

and the ads will disappear

**Reply by Konkss** [reply]
Oh that's excellent! Nice catch.

That's quite buried. Any one else think it should be moved at least next to the DNS over http option?Turning it on Grey's out the DNS option (since it's part of that).

More users might intuitively find it that way.

Either way, happy it already exists in some fashion.
